src/CheckoutSdk/Reconcilation/PayoutStatement.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Checkout.Common;
+using Checkout.Common.Extensions;
+using System;
+
+namespace Checkout.Reconciliation
+{
+    public sealed class PayoutStatement : EquatableResource<PayoutStatement>
+    {
+        public string Id { get; set; }
+
+        public string Status { get; set; }
+
+        public double Currency { get; set; }
+
+        public string PayoutFee { get; set; }
+
+        public string NetAmount { get; set; }
+
+        public string CarriedForwardAmount { get; set; }
+
+        public string CurrentPeriodAmount { get; set; }
+
+        public DateTime Date { get; set; }
+
+        public DateTime PeriodStart { get; set; }
+
+        public DateTime PeriodEnd { get; set; }
+
+        public override bool EqualExp(PayoutStatement other)
+            => Id.EqualsNull(other.Id);
```

Comment:
Question regarding the usage of the extension - what happens if `Id` is null?

Reply:
EqualsNull compare even if both are null. The normal Equals if Id is null throw exception.
That's why normally if you use equal: 
  - or do an if (property != null) eachtime and then property.Equal
  - do an extension that manage that sentence.

The other alternative is using Equals(object, object) but as expect object, is not good for comparing primitive objects.